src/Symfony/Component/Intl/Resources/bin/compile

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-[[ $1 == force ]] && docker pull jakzal/php-intl:8.2-73.2
+[[ $1 == force ]] && docker pull jakzal/php-intl:8.3-74.1
```

Comment:
can be removed btw; no need to re-pull tags

Reply:
deleted 👍